services/app/src/static/style/loginScreen.css

```diff
@@ -0,0 +1,8 @@
+.login-screen {
+    background: #f3f6fa;
```

Comment:
Try to pick a single convention and stick to it. Either choose uppercase color codes or lower case.